Automattic/jetpack
# Search: avoid registering the widget when the module is not active

Fixes #21587

#### Changes proposed in this Pull Request:

The `Jetpack_Search` class will not be loaded if the Search module is not active. So let's not register the widget when the feature is not available.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* On a site connected to WordPress.com, where you have a plan but where the Search module is inactive.
* Go to Jetpack > Settings and enable Extra Sidebar Widgets
* Go to Appearance > Widgets
* You should not see any errors.
* Activate the Search feature.
* You should be able to use the Jetpack Search Widget.
